cli.js

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env node
 
 var chalk = require("chalk");
-var lib = require(".");
+var argv = require("yargs").argv;
+var Changelog = require(".").Changelog;
+var ConfigurationError = require(".").ConfigurationError;
 
-var Changelog = lib.Changelog;
-var ConfigurationError = lib.ConfigurationError;
+if (argv.help) {
```

Comment:
@evocateur so I tried the "positional arguments approach". It's kind of a workaround because `yargs` supports that only for _commands_ and because if we want them to be `<fromTag>..<toTag>` it's considered as 1 argument (because of the `..`) and we need to do some parsing anyway.

Those will be the changes. I'll keep them stashed locally until we decide whether to go with that or not.

```diff
diff --git cli.js cli.js
index 841e66a..ae2d5c0 100755
--- cli.js
+++ cli.js
@@ -1,39 +1,57 @@
 #!/usr/bin/env node
-
 var chalk = require("chalk");
 var Changelog = require(".").Changelog;
 var ConfigurationError = require(".").ConfigurationError;

 var argv = require("yargs")
-  .usage("Usage: lerna-changelog [options]")
-  .options({
-    "tag-from": {
-      type: "string",
-      desc: "A git tag that determines the lower bound of the range of commits (defaults to last available)"
-    },
-    "tag-to": {
-      type: "string",
-      desc: "A git tag that determines the upper bound of the range of commits"
-    }
-  })
+  .usage("Usage: lerna-changelog <fromTag>..<toTag>")
   .example(
     "lerna-changelog",
     "create a changelog for the changes after the latest available tag"
   )
   .example(
-    "lerna-changelog --tag-from 0.1.0 --tag-to 0.3.0",
+    "lerna-changelog 0.1.0",
+    "create a changelog for the changes after the given tag"
+  )
+  .example(
+    "lerna-changelog 0.1.0..0.3.0",
     "create a changelog for the changes in all tags within the given range"
   )
   .version()
   .help()
   .argv;

+var fromTag;
+var toTag;
+var args = argv._;
+if (args.length > 1) {
+  console.error(
+    chalk.red("Invalid arguments, must be in format `<fromTag>..<toTag>`")
+  );
+  process.exit(1);
+} else if (args.length === 1) {
+  var indexOfSeparator = args[0].indexOf("..");
+  if (indexOfSeparator === 0) {
+    console.error(
+      chalk.red("The lower bound tag is required `<fromTag>..<toTag>`")
+    );
+    process.exit(1);
+  } else if (indexOfSeparator === -1) {
+    fromTag = args[0];
+  } else {
+    var parts = args[0].split("..");
+    fromTag = parts[0];
+    toTag = parts[1];
+  }
+}
+
+var options = { fromTag: fromTag, toTag: toTag };
 try {
-  console.log((new Changelog(argv)).createMarkdown());
+  console.log(new Changelog(options).createMarkdown());
 } catch (e) {
   if (e instanceof ConfigurationError) {
     console.log(chalk.red(e.message));
   } else {
-    throw (e);
+    throw e;
   }
 }
```